src/Apple/Provider.php

```diff
@@ -168,6 +220,25 @@ public static function verify($jwt)
         throw new InvalidStateException('Invalid JWT Signature');
     }
 
+    /**
+     * Verify Apple jwt via static function.
+     *
+     * @param string $jwt
+     *
+     * @return bool
+     *
+     * @see https://appleid.apple.com/auth/keys
+     */
+    public static function verify($jwt)
+    {
+        return (new self(
+            new Request(),
+            config('services.apple.client_id'),
```

Comment:
```suggestion
            $this->getConfig('client_id', '')
```

And other calls to `config()`

Reply:
You can call $this-> in static function, or ?

Reply:
oh my bad, didn't realise it was static. Is that why it overrides the global config above?

Reply:
> oh my bad, didn't realise it was static. Is that why it overrides the global config above?

yes